spring-integration-core/src/test/java/org/springframework/integration/dsl/routers/RouterTests.java

```diff
@@ -613,6 +614,10 @@ public void testNestedScatterGatherSuccess() {
 
 	}
 
+	@Autowired
+			@Qualifier("scatterGatherWiretapChannel")
```

Comment:
Something is wrong with your formatting.

Reply:
corrected format